app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -26,6 +26,9 @@ export interface IPublishRepositorySettings {
   /** The repository's description. */
   readonly description: string
 
+  /** Whether the current user (or organization if chosen by user) is on a non-paid plan */
+  readonly canCreatePrivateRepo: boolean
```

Comment:
@shiftkey Any naming suggestions?

Reply:
`hasPaidPlan`?

Reply:
I believe an organization can have a paid, but not allow members to create private repositories.

Reply:
@damaneice no strong feelings here about a better name